src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,53 @@
+package seedu.addressbook.common;
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+    
+    /*
+     *  Tests isAnyNull method
+     */
+    @Test
+    public void testIsAnyNull_noNull() {
```

Comment:
The standard format is methodYouAreTesting_ScenarioYouAreTesting_ExpectedValue 
i.e., AnyNull_emptyList_falseReturned